lib/src/chip/chip.component.html

```diff
@@ -0,0 +1,3 @@
+<div [ngClass]="chipType" [style.cursor]="chipCursor" [style.display]="chipDisplay">
```

Comment:
I wouldn't use a style attribute since that takes styling away from the user of the component. This should be defined in the chip.component.scss